plugins/executor.js

```diff
@@ -18,6 +19,7 @@ const SCHEMA_PIPELINE = Joi.object().keys({
 
 const SCHEMA_START = Joi.object().keys({
     build: Joi.object(),
+    causeMessage,
```

Comment:
does this need DB change ?

Reply:
it's not changing a model, so I don't think so
just adding a param to what the executor is expecting for the `start()`